src/CodeGenerator/Program.cs

```diff
@@ -360,52 +411,65 @@ static void Main(string[] args)
                 writer.WriteLine(string.Empty);
                 writer.PushBlock($"namespace {projectNamespace}");
                 writer.PushBlock($"public static unsafe partial class {classPrefix}");
-                foreach (FunctionDefinition fd in defs.Functions)
+                EmitFunctions(false);
+                writer.PopBlock();
+                writer.PopBlock();
+
+                if (useInternals)
                 {
-                    if (TypeInfo.SkippedFunctions.Contains(fd.Name)) { continue; }
+                    writer.PushBlock($"namespace {projectNamespace}{InternalNamespace}");
+                    writer.PushBlock($"public static unsafe partial class {classPrefix}");
+                    EmitFunctions(true);
+                    writer.PopBlock();
+                    writer.PopBlock();
+                }
 
-                    foreach (OverloadDefinition overload in fd.Overloads)
```

Comment:
This code was promoted to a local function so it could run twice, with the internal flag set or not. The code is the same except for line 433. Originally this method does a ton of processing on the member data only to do a final check, if it isn't a member function (and thus belongs on the `ImGui` class) then returns. I'm unsure why that was done at the end of all the logic, so I moved it here to just filter them out at the start. Now this loop ONLY processes non-member functions.